app/data/ballot-plaintext-simple.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I'm happy to put these files elsewhere, but working files for the `1.2.2` version of electionguard are hard to find and I need them somewhere in source control, ideally in the python-api or web project.  They're listed in the instructions of how to test this PR, and I hope to use them in unit tests.

Reply:
root didn't feel quite right, moved to `/test/integration/data` where there are some other similar files